### ISSUE TITLE
Null Pointer Exception with DateTimeDomain

### ISSUE THREAD

**Issue by vmolina** [jpmml/sklearn2pmml#213]
Whenever trying to buid the PMML from the PMMLPipeline object in python i have the a Null pointer exception.
```python
 pipe = PMMLPipeline([
        ("transformations", DataFrameMapper([            
            (["time_string", ], [DateTimeDomain(),DaysSinceYearTransformer(year=2018)]),
        ])) 
    ])
pipe.fit_transform(x_train, y_train)
sklearn2pmml(pipe, "test_pipe.pmml")
```
This outputs the following issue.

> Standard output is empty

> Standard error:
> Mar 25, 2020 4:56:26 PM org.jpmml.sklearn.Main run
> INFO: Parsing PKL..
> Mar 25, 2020 4:56:26 PM org.jpmml.sklearn.Main run
> INFO: Parsed PKL in 45 ms.
> Mar 25, 2020 4:56:26 PM org.jpmml.sklearn.Main run
> INFO: Converting..
> Mar 25, 2020 4:56:26 PM org.jpmml.sklearn.Main run
> SEVERE: Failed to convert
> java.lang.NullPointerException
> 	at org.jpmml.sklearn.CalendarUtil.toLocalDateTime(CalendarUtil.java:43)
> 	at sklearn2pmml.decoration.Domain.standardizeValue(Domain.java:253)
> 	at sklearn2pmml.decoration.Domain.encodeFeatures(Domain.java:107)
> 	at sklearn2pmml.decoration.TemporalDomain.encodeFeatures(TemporalDomain.java:46)
> 	at sklearn.Transformer.updateAndEncodeFeatures(Transformer.java:118)
> 	at sklearn_pandas.DataFrameMapper.initializeFeatures(DataFrameMapper.java:73)
> 	at sklearn.Initializer.encodeFeatures(Initializer.java:44)
> 	at sklearn.Transformer.updateAndEncodeFeatures(Transformer.java:118)
> 	at sklearn.Composite.encodeFeatures(Composite.java:129)
> 	at sklearn2pmml.pipeline.PMMLPipeline.encodePMML(PMMLPipeline.java:208)
> 	at org.jpmml.sklearn.Main.run(Main.java:228)
> 	at org.jpmml.sklearn.Main.main(Main.java:148)
> 
> Exception in thread "main" java.lang.NullPointerException
> 	at org.jpmml.sklearn.CalendarUtil.toLocalDateTime(CalendarUtil.java:43)
> 	at sklearn2pmml.decoration.Domain.standardizeValue(Domain.java:253)
> 	at sklearn2pmml.decoration.Domain.encodeFeatures(Domain.java:107)
> 	at sklearn2pmml.decoration.TemporalDomain.encodeFeatures(TemporalDomain.java:46)
> 	at sklearn.Transformer.updateAndEncodeFeatures(Transformer.java:118)
> 	at sklearn_pandas.DataFrameMapper.initializeFeatures(DataFrameMapper.java:73)
> 	at sklearn.Initializer.encodeFeatures(Initializer.java:44)
> 	at sklearn.Transformer.updateAndEncodeFeatures(Transformer.java:118)
> 	at sklearn.Composite.encodeFeatures(Composite.java:129)
> 	at sklearn2pmml.pipeline.PMMLPipeline.encodePMML(PMMLPipeline.java:208)
> 	at org.jpmml.sklearn.Main.run(Main.java:228)
> 	at org.jpmml.sklearn.Main.main(Main.java:148)
> 
> 
> ---------------------------------------------------------------------------
> RuntimeError                              Traceback (most recent call last)
> <ipython-input-33-f0e2055d1839> in <module>
> ----> 1 sklearn2pmml(pipe, "test_pipe.pmml")
> 
> ~/PycharmProjects/edo-ds-meta-gap/venv/lib/python3.7/site-packages/sklearn2pmml/__init__.py in sklearn2pmml(pipeline, pmml, user_classpath, with_repr, debug, java_encoding)
>     264                                 print("Standard error is empty")
>     265                 if retcode:
> --> 266                         raise RuntimeError("The JPMML-SkLearn conversion application has failed. The Java executable should have printed more information about the failure into its standard output and/or standard error streams")
>     267         finally:
>     268                 if debug:
> 
> RuntimeError: The JPMML-SkLearn conversion application has failed. The Java executable should have printed more information about the failure into its standard output and/or standard error streams

pandas==1.0.2
sklearn-pandas==1.8.0
sklearn2pmml==0.55.3
scikit-learn==0.21.3

Hope you can help here.

**Comment by vruusmann** [comment]
A solid programming mistake by me.

As a temporary workaround, you could specify a non-null missing value replacement value for the "time_string" column:

```python
mapper = DataFrameMapper([
  ("time_string", [DateTimeDomain(missing_value_replacement = "1960-01-01T00:00:00")])
])
```

**Comment by vruusmann** [comment]
A fixed version is available in PyPI as SkLearn2PMML 0.55.4.

**Comment by vmolina** [comment]
Worked smoothly 
Thank You!
